Formula/aften.rb

```diff
@@ -13,10 +13,24 @@ class Aften < Formula
 
   depends_on "cmake" => :build
 
+  resource "sample_wav" do
+    url "http://www.mediacollege.com/audio/tone/files/1kHz_44100Hz_16bit_05sec.wav"
+    sha256 "949dd8ef74db1793ac6174b8d38b1c8e4c4e10fb3ffe7a15b4941fa0f1fbdc20"
+  end
+
   def install
     mkdir "default" do
       system "cmake", "-DSHARED=ON", "..", *std_cmake_args
       system "make", "install"
     end
+
+    resource("sample_wav").stage { bin.install "1kHz_44100Hz_16bit_05sec.wav" }
+  end
+
+  test do
+    assert_match %r{Aften: A\/52 audio encoder}, shell_output("#{bin}/aften -h").lstrip.lines.map(&:chomp).first
+
+    assert_match("output format: 48000 Hz stereo (2/0)",
+      pipe_output("#{bin}/aften 1kHz_44100Hz_16bit_05sec.wav sample.ac3").lines.map(&:chomp).find { |line| line.start_with? "output" })
```

Comment:
I think just doing:
`system "#{bin}/aften", "#{testpath}/1kHz_44100Hz_16bit_05sec.wav", "sample.ac3"`
is fine (after moving the `resource` staging here into the test block) -- no need to check the output if aften returns a meaningful exit code.